Sources/ParseSwift/Objects/ParseObject.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  ParseObject.swift
+//  ParseSwift
+//
+//  Created by Pranjal Satija on 7/18/20.
```

Comment:
It looks like many of these files were renamed, but the Flo’s name was removed and the original date it was created updated to the current date. Are you creating new files and dropping in the code or renaming them in Xcode? If you are creating new files, I think you are going to have merge conflicts with the other PRs

I think the original creator and dates should be left and if there needs to be additional comments, we can add them. If you create new files with new code, then it should look the way this file looks

An example I had is in PR 15 was URLSession+sync.swift. I had created URLSession+async.swift (brand new file and code), after the bugs were worked out, I deleted URLSession+sync.swift as it wasn’t being used anymore and renamed URLSession+async.swift to URLSession.swift. This case is slightly different, but I’m even thinking of putting Flo’s info back in there.

Reply:
Just to add here... the copyright to Parse should be changed to Parse Community or removed at some point.

And in terms of who's name is in there, GitHub holds all the history so I really don't think it's important / helpful to have names at all?

Reply:
I’m not by my computer to verify, but I don’t believe the file will show the history if it’s created as new with a different name. I think it will still shows if you rename the file, or if a file is deleted and then a new file is created in the future with the same name. For example I don’t think the URLSession.swift file will show the history based on what I mentioned in the previous post. My belief is we should always cite the source and preserve the history when possible.

@TomWFox to your point about the names, I think the Parse-Swift Xcode project is currently setup to automatically input our names, copyright, and date. For example, I haven’t intentionally add my names to any files in the PRs, it was done automatically.

Reply:
@cbaker6 Good catch, I'll put Flo's name back. I don't use the file rename built into Xcode because I've found it to be unreliable sometimes. I usually create a new file, and then copy/paste the body from import statements down. Xcode just automatically inserts my name and copyright at the top. It's never something I really noticed or paid a lot of attention to, but I'll fix it.

@TomWFox Xcode inserts those names and copyrights by default. If it's an issue, I think we can follow the steps listed in [this](https://useyourloaf.com/blog/changing-xcode-header-comment/) article to insert a custom copyright template for this project that includes an abridged license or any other information, along with a copyright owned by Parse Community.

Reply:
@cbaker6 @TomWFox what do you guys think about removing these headers entirely? Git already keeps track of who made what, when they made it, etc. and these headers will quickly get cumbersome to maintain. If nobody has a problem with it, I can configure Xcode not to make them for new files, and I can get rid of the ones that are already there.

Reply:
@cbaker6 @TomWFox just want to mention this again before this PR is merged; if we want, we can save this change for another one down the road.